biome.json

```diff
@@ -26,7 +26,8 @@
         "noGlobalIsNan": "error",
         "noGlobalIsFinite": "error",
         "noPrototypeBuiltins": "error",
-        "noVar": "error"
+        "noVar": "error",
+        "noExportsInTest": "off"
```

Comment:
Now that the tests have describe / it, biome recognises them as tests and complains about the exports